rest_framework_tracking/management/commands/clearapilogs.py

```diff
@@ -0,0 +1,37 @@
+from django.core.management.base import BaseCommand, CommandError
+from rest_framework_tracking.models import APIRequestLog
+from optparse import make_option
+import datetime
+from django.utils import timezone
+
+class Command(BaseCommand):
+    help = 'Removes all api logs OR keeps X number of days of api logs'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--days_num',
+            help='Keep X number of days of logs and delete the rest',
+            type=int,
+            choices=[x + 1 for x in range(7)],
```

Comment:
is there any need to restrict the input to just 7 days? Maybe some people might want to keep last month's/years worth of logs...

Reply:
No need to restrict it to 7 days. Now it is 100,000